src/operator/softmax_output-inl.h

```diff
@@ -88,8 +96,12 @@ class SoftmaxOutputOp : public Operator {
       Tensor<xpu, 2> label = in_data[softmaxout_enum::kLabel].FlatTo2D<xpu, real_t>(s);
       Tensor<xpu, 3> out = out_data[softmaxout_enum::kOut].get_with_shape<xpu, 3, real_t>(s3, s);
       Tensor<xpu, 3> grad = in_grad[softmaxout_enum::kData].get_with_shape<xpu, 3, real_t>(s3, s);
-      SoftmaxGrad(grad, out, label);
-      grad *= param_.grad_scale/s3[2];
+      if (param_.use_ignore) {
+          SoftmaxGrad(grad, out, label, static_cast<real_t>(param_.ignore_label));
+      } else {
+          SoftmaxGrad(grad, out, label);
+      }
+      grad *= param_.grad_scale;
```

Comment:
@piiswrong , sorry for that, i remove it by accidently, could you please recover it, i'm not in my computer today, ths~